### DISCUSSION TITLE
[bug]Japanese conversion is not possible in taglist

### DISCUSSION THREAD

**Discussion by kryushiki** [formkit/formkit#807] [General]
Enter is pressed before converting when entering Japanese
can not convert
https://github.com/formkit/formkit/assets/9635671/251c28f5-2a8b-4edd-b56f-2254cdbd7d3a
https://formkit.com/inputs/taglist

should be like this...
https://github.com/formkit/formkit/assets/9635671/eb53a386-0291-480a-b75d-a428d8d4cefa
https://buefy.org/documentation/taginput

**Comment by sashamilenkovic** [comment]
@kryushiki definitely something worth taking a look at
